taskvine/src/worker/vine_worker.c

```diff
@@ -1191,6 +1192,8 @@ static int process_ready_to_run_now(struct vine_process *p, struct vine_cache *c
 		p->library_process = find_library_for_function(p->task->needs_library);
 		if (!p->library_process)
 			return 0;
+		if (!p->library_process->library_ready)
```

Comment:
This isn't quite right b/c it will find the first library available, and then if it isn't initialized, will fail.

Just adjust `find_library_for_function` to check for `library_ready` as well as other things.